editor/modes/visual-editor/block-list.js

```diff
@@ -164,7 +172,12 @@ class VisualEditorBlockList extends wp.element.Component {
 
 		return (
 			<div>
-				{ blocksWithInsertionPoint.map( ( uid ) => {
+				{ ! blocks.length && (
+					<button className="editor-visual-editor__placeholder" onClick={ this.appendDefaultBlock }>
```

Comment:
I guess we need both because Firefox and Safari don't focus on click in MacOS. Or better transform this to an input